go/base-arm/Dockerfile.tmpl

```diff
@@ -57,12 +57,12 @@ RUN \
             libsqlite3-0 \
         && rm -rf /var/lib/apt/lists/*
 
-ARG GOLANG_VERSION=1.24.2
+ARG GOLANG_VERSION=1.24.3
```

Comment:
@v1v I believe the tooling for the automated version bump has a bug. The `GOLANG_VERSION` is the same for the stdlib go and the microsoft go download, but the automation only updated the `MSFT_DOWNLOAD_SHA256`. The `GOLANG_DOWNLOAD_SHA256` (line 69) also needs to be updated though, otherwise the version bump for stdlib go is failing. 
Same note for the non-arm file.

Reply:
Uhm, let me revisit, because I know there are two updates:
- traditional golang
- Microsoft one.

and changes were partially done:
- https://github.com/elastic/golang-crossbuild/actions/runs/14874392357/job/41768847694#step:3:366